tipg/dbmodel.py

```diff
@@ -309,23 +319,42 @@ def _select(
     def _select_mvt(
         self,
         properties: Optional[List[str]],
-        geometry_column: GeometryColumn,
+        geometry_column: Column,
         tms: TileMatrixSet,
         tile: Tile,
     ):
         bbox = tms.xy_bounds(tile)
+        llbounds = tms.bounds(tile)
         tms_srid = tms.crs.to_epsg()
+
+        if (
+            tms.identifier in ("WebMercatorQuad")
+            and llbounds.top > 85
+            or llbounds.bottom < -85
+        ):
+            geom = logic.Func(
+                "st_intersection",
+                logic.Func("st_makeenvelope", -180, 85, 180, -85, 4326),
+                logic.Func(
+                    "st_transform",
+                    logic.V(geometry_column.name),
+                    pg_funcs.cast(4326, "int"),
+                ),
```

Comment:
Can you explain what's going on here?

Reply:
It clips geometries at +-85 so that things don't error near the poles when converting to 3857.

Reply:
I'm not sure what I was doing was either right or the best thing to do.

We can just skip doing anything for now, but I do think that we are going to want to do something for this since for a number of examples we end up getting all kinds of errors that bubble up from postgis when we do nothing. The approach I took here was really playing whack-a-mole with those problems rather than anything well thought out.